src/test/run-make-fulldeps/remap-path-prefix/auxiliary/lib.rs

```diff
@@ -0,0 +1,3 @@
+fn lib() {
```

Comment:
Can you make this function `pub` so that we can be sure that the compiler actually generates it?